airbyte-integrations/connectors/source-mailjet-sms/source_mailjet_sms/spec.yaml

```
@@ -0,0 +1,25 @@
+documentationUrl: https://dev.mailjet.com/sms
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Mailjet Sms Spec
+  type: object
+  required:
+    - token
+  additionalProperties: true
+  properties:
+    token:
+      type: string
+      description: >- 
+        Your access token. See <a
+        href="https://dev.mailjet.com/sms/reference/overview/authentication">here</a>.
+      airbyte_secret: true
```

```suggestion
    token:
      title: Access Token
      type: string
      description: >- 
        Your access token. See <a
        href="https://dev.mailjet.com/sms/reference/overview/authentication">here</a>.
      airbyte_secret: true
```
Please add title to all parameters.